SpatialGDK/Source/SpatialGDK/Private/SpatialView/ViewDelta.cpp

```diff
@@ -291,7 +291,7 @@ void ViewDelta::ProcessOp(Worker_Op& Op)
 		ComponentChanges.Emplace(Op.op.component_update);
 		break;
 	default:
-		checkNoEntry();
+		break;
```

Comment:
Because we 0 out the op type as a hack to stop things being processed twice. Can leave it as a separate list like it used to be as this is trying to be clever.